data/json/itemgroups/guns.json

```diff
@@ -4,6 +4,7 @@
     "id": "guns_pistol_common",
     "//": "Pistols commonly owned by citizens and found in many locations.",
     "items": [
+      [ "glock_17", 75 ],
```

Comment:
This makes the already incredibly common Glock family about 50% more common, steal some of the spawn chance from the other Glocks instead of increasing it.

Reply:
Done in commit #8f73d2b